install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -625,6 +625,36 @@ spec:
         resources:
           {{- toYaml . | trim | nindent 10 }}
         {{- end }}
+      {{- if .Values.waitForKubeProxy }}
+      - name: wait-for-kube-proxy
+        image: {{ include "cilium.image" .Values.image | quote }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+          - bash
+          - -c
+            - |
+              while true
+              do
+                if iptables-nft-save -t mangle | grep -E '^:(KUBE-IPTABLES-HINT|KUBE-PROXY-CANARY)'; then
```

Comment:
Is finding one or the other enough? I'm assuming the rules are atomically set at the same time?